src/Clan.ts

```diff
@@ -245,4 +298,73 @@ export class Clan {
     );
     return result.includes("You contributed");
   }
+
+  /**
+   * Take items from the stash
+   * 
+   * This function will also take equivalent foldables if the original item cannot be found
+   * 
+   * @param items Items to take
+   * @returns Items successfully taken
+   */
+  @validate
+  take(items: Item[]): Item[] {
```

Comment:
Is there a non-terrible way to set this up to let us take multiple of an item/fold group? Right now, because we skip if we have it, putting the same item in the array multiple times does nothing.

Reply:
Yeah I noticed that, it's also an issue with the garbo implementation. I can definitely do it as a map or entries-style array